src/JustEat.StatsD/StatsDMessageFormatter.cs

```diff
@@ -12,15 +12,14 @@ public class StatsDMessageFormatter
         [ThreadStatic]
         private static Random _random;
 
-        private readonly CultureInfo _cultureInfo;
+        private static readonly CultureInfo CultureInfo = CultureInfo.InvariantCulture;
```

Comment:
Very minor comment, I'd name this `InvariantCulture`, or something to that effect